### DISCUSSION TITLE
Transmission GUI does not work

### DISCUSSION THREAD

**Discussion by Ircama** [Freetz-NG/freetz-ng#930] [Allgemein] [answered]
Transmission GUI does not work because the content of the Web-Home Directory is not including *index.html*.

```
root@fritz:/var/mod/root# ls /usr/share/transmission-web-home
README.md            package.json         stylelint.config.js
assets               prettier.config.js   webpack.config.js
babel.config.js      public_html
package-lock.json    src
```

The content of this directory should be instead the one of the *public_html* subfolder:

```
root@fritz:/var/mod/root# ls /usr/share/transmission-web-home/public_html
images                         transmission-app.js
index.html                     transmission-app.js.LEGAL.txt
transmission-app.css
```

Anyway, an option could be to replace everything with [the latest release](https://github.com/transmission-web-control/transmission-web-control/releases/) of [transmission-web-control](https://github.com/transmission-web-control/transmission-web-control).

**Comment by fda77** [comment]
i dont add things for programm i dont use, and nobody else too

**Reply by Ircama** [reply]
OK, just mentioning that the standard setup is wrong with /usr/share/transmission-web-home that needs to include the content of /usr/share/transmission-web-home/public_html. Alternatively, `Web-Home Directory: ` should point to that directory instead of `/usr/share/transmission-web-home`.

**Reply by fda77** [reply]
I've no idea because i never used this programm/daemon/whatever

**Reply by Ircama** [reply]
Would a PR be useful?

**Comment by fda77** [comment]
Im not sure what you writing about. transmission is broken and could no be used at all? Because of wrong web files?

**Reply by Ircama** [reply]
transmission is made of more components. transmission-daemon works, while transmission-web is broken and cannot be used at all because of the wrong web files. If you attempt to access http://fritz.box:9091/, which redirects to http://fritz.box:9091/transmission/web/, you receive the message "fritz.box refused to connect." If you fix the web files removing the content of /usr/share/transmission-web-home and using instead the one in /usr/share/transmission-web-home/public_html, it works.

**Comment by fda77** [comment]
About what are you writing?
Version 3?
Version 4?
Package transmission?
Package transmission-cgi?

**Comment by Ircama** [comment]
> About what are you writing? Version 3? Version 4?

I merely performed the freetz-ng installation.

```
  --- Transmission 3.00/4.0.5 (binary only)                                                                         │ │
  │ │                                                 Version (4.0.5)  --->                                                                                       │ │
  │ │                                                 SSL library (OpenSSL)  --->                                                                                 │ │
  │ │                                           [*]   Transmission client                                                                                         │ │
  │ │                                           [X]   Transmission daemon                                                                                         │ │
  │ │                                           [*]     Transmission remote                                                                                       │ │
  │ │                                           [*]     Transmission web interface                                                                                │ │
  │ │                                           [*]   transmission-create                                                                                         │ │
  │ │                                           [*]   transmission-edit                                                                                           │ │
  │ │                                           [*]   transmission-show                                                                                           │ │
  │ │                                           [*]   Select Transmission CGI
```

> Package transmission?

```
root@fritz:/var/mod/root# transmission-daemon -V
transmission-daemon 4.0.5 (a6fe2a64aa)
```

> Package transmission-cgi?

Freetz about.cgi: Transmission 4.0.5

**Comment by fda77** [comment, marked as answer]
I added v4 months ago for some people and they tested it - and it seems they where not able to tell what works and what not (what is a general problem for freetz).

7c8a587a0

run "make transmission-recompile" to apply after pull

**Reply by Ircama** [reply]
I confirm that commit https://github.com/Freetz-NG/freetz-ng/commit/7c8a587a0ecee8aaa8428a5ba0f80bf0a53574ae fixed the issue. Thanks a lot!
